cli/command/service/opts.go

```diff
@@ -291,8 +308,55 @@ func (opts *healthCheckOptions) toHealthConfig() (*container.HealthConfig, error
 	return healthConfig, nil
 }
 
+func parsePortCSV(value string) (map[string]string, error) {
+	kvPair := make(map[string]string)
```

Comment:
Please make this a flag type like `MountOpt`. It will make the code below a lot cleaner when you go to read the values.

It also means that we validate during flag parsing instead of waiting until converting the types.